app/experimenter/experiments/models/nimbus.py

```diff
@@ -270,7 +270,10 @@ def allocate_bucket_range(self):
         )
 
     def can_review(self, reviewer):
-        if self.publish_status == NimbusExperiment.PublishStatus.REVIEW:
+        if self.publish_status in (
+            NimbusExperiment.PublishStatus.REVIEW,
+            NimbusExperiment.PublishStatus.WAITING,
```

Comment:
Why change this?  When it's in waiting it can no longer be reviewed?

Reply:
Doesn't this change cover [this state](https://storage.googleapis.com/mozilla-storybooks-experimenter/commits/ba828eef86092c0cb5b77a7579f3c6bcef68fdfe/nimbus-ui/index.html?path=/story/pages-requestreview-exp-1143-end-experiment-review-refactor--review-pending-in-remote-rettings-user-can-review); isn't it in `WAITING` there?

Reply:
Yeah but you can't review from experimenter there, that's already done.

Reply:
It *can* be reviewed, though - in Remote Settings: https://storage.googleapis.com/mozilla-storybooks-experimenter/commits/ba828eef86092c0cb5b77a7579f3c6bcef68fdfe/nimbus-ui/index.html?path=/story/pages-requestreview-exp-1143-end-experiment-review-refactor--review-pending-in-remote-rettings-user-can-review

Reply:
We're using the `canReview` flag to display that pending-in-remote-settings UI

Reply:
Oh I see you're gating showing the RS link on can_review because the owner can't do that either. Okay yep that makes sense 👍

Reply:
That slightly changes the intention of the method here:

https://github.com/mozilla/experimenter/blob/a892d6fbead11126894beac7f9b5ec665f75a92c/app/experimenter/experiments/api/v5/serializers.py#L356-L360

But maybe the publish_status transition filtering will be sufficient since the user can't move it out of waiting only the tasks can do that.